code/obj/machinery/computer/security/riot.dm

```diff
@@ -68,13 +68,16 @@
 			if ("help")
 				if (!signal.data["topic"])
 					returnsignal.data["description"] = "Armory Authorization Computer - allows for lowering of armory access level to SECURITY. Wireless authorization requires NETPASS_HEADS"
-					returnsignal.data["topics"] = "authorize"
+					returnsignal.data["topics"] = "authorize and unauthorize"
```

Comment:
```suggestion
					returnsignal.data["topics"] = "authorize,unauthorize"
```

Reply:
Does that acctually do something or is it just because it is easier to read?, already committed the suggestion, asking because I really don't mess with this stuff.

Reply:
It just keeps the syntax in line with the other wireless devices with multiple help topics, but functionally it's the same